plugin.rb

```diff
@@ -8,20 +8,26 @@
 require 'json'
 require File.expand_path('../lib/validators/discourse_slack_enabled_setting_validator.rb', __FILE__)
 
-enabled_site_setting :slack_enabled
-
 PLUGIN_NAME = "discourse-slack-official".freeze
 
+enabled_site_setting :slack_enabled
+
 register_asset "stylesheets/slack_admin.scss"
 
 after_initialize do
 
   unless ::PluginStore.get(PLUGIN_NAME, "not_first_time")
+    id = SecureRandom.hex(16)
```

Comment:
Actually the rows will look alike below
```
{ key: "filter_RANDOMSTRING", value: { channel: "#default", category_id: "2", tags: ["hello", "welcome"], filter: "follow" } }
{ key: "category_2_#default", value: "RANDOMSTRING" }
{ key: "tag_hello_#default", value: "RANDOMSTRING" }
{ key: "tag_welcome_#default", value: "RANDOMSTRING" }
```

It is not just associated with a single category or a single tag.

I am using first row to manage filters on admin console. And other rows to check data while notification jobs.

If you better workaround can you please guide me?